src/App.js

```diff
@@ -0,0 +1,28 @@
+import NavBar from "./components/NavBar";
+import About from "./components/About";
+import Card from "./components/Card";
+import "./styles/common.css";
+import Arr from "./cardData";
+
+function App() {
```

Comment:
use arrow functions

Reply:
read this - https://articles.jalantechnologies.com/best-practices-for-clean-react-apps/